tracer/src/Datadog.Trace/Logging/DirectSubmission/DirectLogSubmissionSettings.cs

```diff
@@ -140,42 +154,51 @@ public DirectLogSubmissionSettings(IConfigurationSource? source, IConfigurationT
                 }
             }
 
-            var isEnabled = enabledIntegrations is not null;
+            // For OTLP logs, automatically enable ILogger integration
+            if (otlpLogsEnabled)
+            {
+                enabledIntegrations ??= new bool[IntegrationRegistry.Ids.Count];
+                enabledIntegrations[(int)IntegrationId.ILogger] = true;
+            }
+
             _enabledIntegrations = enabledIntegrations;
 
+            var isEnabled = enabledIntegrations is not null;
+
             if (!AzureFunctionsHostEnabled && EnvironmentHelpers.IsRunningInAzureFunctionsHost())
             {
-                // If we are in `dotnet-isolated` Azure Functions host customers may want to disable direct log submission
-                // because the host will re-log logs coming from the worker process causing duplicate logs.
                 isEnabled = false;
             }
 
-            if (string.IsNullOrWhiteSpace(Host))
+            if (!otlpLogsEnabled)
```

Comment:
Are there any validations we need to do for OTLP Logs?

Reply:
Since we don't require a specific Site or API and have defaults instead I think we should be good with what we have within TracerSettings.cs